plenum/common/util.py

```diff
@@ -507,6 +508,17 @@ def is_network_ip_address_valid(ip_address):
         return True
 
 
+def is_hostname_valid(hostname):
```

Comment:
Can we use `socket.gethostbyname`?

Reply:
The problem is `gethostbyname` tries to resolve the name, the dev while setting it up might not have created the host. Moreover it gives IPv4.